intranet/apps/eighth/models.py

```diff
@@ -332,6 +335,7 @@ def restricted_activities_available_to_user(cls, user):
 
     @classmethod
     def available_ids(cls):
+        """Returns all available IDs for which there is no EighthActivity with that ID."""
```

Comment:
Needs return type and `Returns` block
```suggestion
        """Returns all available IDs not used by an EighthActivity."""
```